docroot/sites/default/default.services.yml

```diff
@@ -0,0 +1,174 @@
+parameters:
```

Comment:
we don't want docroot/sites/default/default.services.yml and docroot/sites/default/services.yml